### PR TITLE
Fix overlapping views

### PR DESCRIPTION
**Pull request by Naveen3Singh**

### CODE REVIEW

File: app/src/main/res/values-xxxhdpi/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <dimen name="dialpad_button_size_small">50dp</dimen>
+    <dimen name="incoming_call_arrow_size">50dp</dimen>
+    <dimen name="incoming_call_button_size">72dp</dimen>
+    <dimen name="call_notification_button_size">40dp</dimen>
+    <dimen name="incoming_call_avatar_size">120dp</dimen>
+
+    <dimen name="dialpad_text_size">30sp</dimen>
+    <dimen name="caller_name_text_size">34sp</dimen>
+    <dimen name="call_status_text_size">20sp</dimen>
+</resources>
```
Comment (tibbi) [inline app/src/main/res/values-xxxhdpi/dimens.xml:14]: oh cmon, avoid copypasting all these files, it is a road to hell

Comment (Naveen3Singh) [reply app/src/main/res/values-xxxhdpi/dimens.xml:14]: you mean to not duplicate the same sizes in different dimens.xml files, right? 
or
do you want to keep using a single dimension file for all screen sizes?

I increased the text sizes for sw480dp and sw720dp a little bit

Comment (Naveen3Singh) [reply app/src/main/res/values-xxxhdpi/dimens.xml]: removed duplicated stuff

**Comment by tibbi** [comment]
thats still too many new files, we dont need to scale it that much. It wont be maintainable

**Comment by Naveen3Singh** [comment]
in that case, I think it would be easier to just reduce the button size for every screen like it was before (as we need to create at least two files (fohdpi, xhdpi) to keep the button size large on xhdpi+ devices).

Here's how it looks on my phone with normal density:


We could restructure the layout a bit to avoid overlap but it won't help much with small screens

**Comment by Naveen3Singh** [comment]
Even with the button size reduced, things would still overlap on something like the Nexus S:


but I doubt today's phones come with that configuration

**Comment by tibbi** [comment]
looks good, thanks

**Pull request merged**